cluster-autoscaler/cloudprovider/oci/README.md

```diff
@@ -123,6 +134,7 @@ use-instance-principals = true
 ### Configuration via environment-variables:
 
 - `OCI_USE_INSTANCE_PRINCIPAL` - Whether to use Instance Principals for authentication rather than expecting an OCI config file to be mounted in the container. Defaults to false.
+- `OCI_USE_WORKLOAD_IDENTITY` - Whether to use Workload Identity for authentication. Setting to `true` takes precedence over `OCI_USE_INSTANCE_PRINCIPAL`. When using this flag, the `OCI_RESOURCE_PRINCIPAL_VERSION` (1.1 or 2.2) and `OCI_RESOURCE_PRINCIPAL_REGION` also need to be set. See this [blog post](https://blogs.oracle.com/cloud-infrastructure/post/oke-workload-identity-greater-control-access#:~:text=The%20OKE%20Workload%20Identity%20feature,having%20to%20run%20fewer%20nodes.) for more details on setting the policies for this auth mode.
```

Comment:
> https://blogs.oracle.com/cloud-infrastructure/post/oke-workload-identity-greater-control-access#:~:text=The%20OKE%20Workload%20Identity%20feature,having%20to%20run%20fewer%20nodes.

I think there's some extra cruft at the end of this url that only works for chrome (the #:~: part)

Reply:
That works fine for me in FireFox too. Does it somehow fail in specific browsers? it should just be ignored.

Reply:
Yeah, it gets ignore and doesn't cause failures. I just noticed that it didn't work in Safari when I tried it. It's fine to leave it in as is. I didn't know if it was purposefully in or not.